Gemfile

```diff
@@ -61,6 +61,7 @@ gem 'will_paginate', '>= 3.0.6'
 gem 'will_paginate-bootstrap', '>= 1.0.1'
 gem 'georuby', '2.0'
 gem 'geokit-rails'
+gem 'json'
```

Comment:
Oops, this Gemfile is still sneaking back in. Isn't Gemfile part of your .gitignore file? How's it keep getting included? 

Thanks, StellaMaris!